Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM fedora:29 as build
```

Comment:
I prefer to use more smaller images like alpine.
nginx:alpine or node-static is better because this images provides just static html pages.

Reply:
@ashphy  In a multi stage build the first only the last FROM counts fedora is used as “build” environment in a second FROM files from ./dist/ are copied to document root of a plain httpd container, maybe I can use httpd-alpine for the “second” FROM

Reply:
Ping @ashphy , I changed the base image to nginx-alpine in the last commit:

https://github.com/ashphy/jsonpath-online-evaluator/pull/13/commits/0a35ef2fd02e0ee9356eee2557d6307f4b96828f